src/components/CreateResetPassword/CreateResetPassword.js

```diff
@@ -128,25 +158,43 @@ class CreateResetPassword extends Component {
                 <Row center="xs">
                   <Col xs={6}>
                     <FieldLabel htmlFor="new-password">
-                      <FormattedMessage id={`${this.translateNamespace}.newPassword`} />
+                      <FormattedMessage id={`${this.translationNamespaces.page}.newPassword`} />
                     </FieldLabel>
                   </Col>
                 </Row>
-                <Row center="xs">
-                  <Col xs={6}>
-                    <Field
+                <Row
+                  center="xs"
+                  end="sm"
+                >
+                  <Col
+                    xs={6}
+                    sm={9}
+                  >
+                    <this.newPasswordField
                       id="new-password"
-                      component={TextField}
                       name="newPassword"
+                      autoComplete="new-password"
+                      component={PasswordStrength}
                       type={passwordType}
-                      marginBottom0
-                      fullWidth
+                      username={username}
                       inputClass={styles.input}
-                      validationEnabled={false}
                       hasClearIcon={false}
-                      autoComplete="new-password"
-                      ref={this.newPassword}
-                      withRef
+                      errors={errors}
+                      autoFocus
+                      marginBottom0
+                      fullWidth
+                      token={token}
+                      inputColProps={{
```

Comment:
Please define this `props` in `constructor` so the new object is not being created upon every render. Pay attention to spacing issue inside an object.